lib/linguist/languages.yml

```diff
@@ -4879,6 +4879,14 @@ Shen:
   tm_scope: source.shen
   ace_mode: text
   language_id: 348
+Skript:
+  type: programming
+  color: "#dbbd61"
+  extensions:
+  - ".sk"
+  tm_scope: none
+  ace_mode: text
+  language_id: 14101512
```

Comment:
This ID is not correct. You shouldn't be picking numbers at random or that appear to be in sequence of unused as this can cause conflicts within the languages table on GitHub.com. Please remove the entire line and run `script/update-ids` to generate the correct ID.  This would have been done automatically if you'd use `script/add-language` to add the language.